### DISCUSSION TITLE
Idea to Inertia without for reload, inertia-link and each other methods

### DISCUSSION THREAD

**Discussion by Rene-Roscher** [inertiajs/inertia#909] [Ideas]
Hello all,

I have an idea, so you can save data traffic in your application.

There are props, which may not always need to be reloaded, such as the user or the languages (Only for example). There a parameter like "without" would be a very helpful thing to exclude things from a reload or a request.

For example.
Inertia.reload({ without: ['user', 'language'] });

**Comment by ajnsn** [comment]
If I understand you correctly, [this issue](https://github.com/inertiajs/inertia/issues/377) is propably related to your request?
